showpass/Showpass-E2E-tests
# Add test scripts to verify the Home page element appearance

**Description:**
Create a script to verify the ‘Home’ page overall appearance. 
The following elements need to be verified:
- Top bar elements
- Showpass logo
- En/Fr toggle
- Create Account button
- Login button
- Goole Play and App Store 
- Bottom bar elements

**Jira ticket:**
https://showpass.atlassian.net/browse/TA-7

**How to test:**
Run the following test suite and observe the results: 
/ShowpassE2E/cypress/e2e/home/home-page-appearance.cy.js

**Test execution results:**
<img width="1699" alt="Test Results - homePageAppearance" src="https://github.com/showpass/Showpass-E2E-tests/assets/138615449/42bdb16e-2ba0-4646-9e0c-692777652c41">
